js/src/Ice/ConnectionI.js

```diff
@@ -1140,9 +1154,17 @@ export class ConnectionI {
         return true;
     }
 
+    /**
+     * Sends the next message in the send queue, if any, and returns an array of messages that have already been sent and
```

Comment:
> returns an array of messages that have already been sent and for which a reply has been received.

Message is the terminology we use for Frame in Ice. In this particular case, the messages are necessarily outgoing requests. I would make this clearer in the doc-comment, like:

> returns an array of messages that hold outgoing requests for which we've received replies.

Also, to be clear, do we include / not include oneway requests in this list?